MyWeight/AddViewController.swift

```diff
@@ -125,13 +125,13 @@ class AddViewController: UIViewController, UIPickerViewDataSource, UIPickerViewD
         return weight
     }
 
-    func saveMass() -> Void
+    @objc private func saveMass()
```

Comment:
Since we added the `private` access notation, this func becomes not visible to Objective-C. We use the saveMass as a `#selector` to the target/action patter on a UIButton, which is an Objective-C API, so it needs to be visible.

The solution can be either removing the `private` or adding the `@objc` notation., I prefer to have the additional `@objc` and make sure that things are correctly encapsulated.